dev/lock.yml

```diff
@@ -6,17 +6,21 @@ dev:
   lines:
     rust-toolchain.toml:
       (channel = ").+("): RUST_VERSION
+    ghjk.ts:
+      (const PROTOC_VERSION = ").*(";): PROTOC_VERSION
```

Comment:
To avoid any risk of never updating the version (eg. forgot space before "1.2.3"), I suggest something more fuzzy like `(const\s+PROTOC_VERSION\s+=\s+").*(";)`. Looks pretty awful but we never know.